### DISCUSSION TITLE
How to align radio options vertically?

### DISCUSSION THREAD

**Discussion by eisthf** [google/mesop#735] [Q&A] [answered]
I would like to align radio options vertically. For example, I would like to have:
Radio Group
  [ x ] Option 1
  [   ] Option 2

rather than:
Radio Group
  [ x ] Option 1  [  ] Option 2

**Comment by richard-to** [comment, marked as answer]
I haven't verified, but I think you can do something like this (i.e. add `style=me.Style(display="flex", flex_direction="column")`):

```python
   me.radio(
        options=[
            me.RadioOption(label=value, value=value)
            for value in values
        ],
        style=me.Style(display="flex", flex_direction="column"),
        on_change=on_change_radio,
    )
```
